### DISCUSSION TITLE
5a4fe6d7-4df6-4baf-848a-6645c2ef549a

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9630] [Rules Discussions]
# 5a4fe6d7-4df6-4baf-848a-6645c2ef549a

Customized versions of classes or components should reflect the customization in the name.

https://www.ssw.com.au/rules/use-meaningful-modifiers/

**Comment by jernejk** [comment]
I agree for the most part but `WriteOnlyDbContext` still doesn't tell for what DB it's used. Working on many projects with multiple DB scopes, `{{ MODULE_NAME }}DbContext` (`MODULE_NAME` is the name of the module) is an important naming convention so that you know with what DB you're interacting with.
